op-batcher/batcher/driver.go

```diff
@@ -566,38 +603,88 @@ func (l *BatchSubmitter) throttlingLoop(wg *sync.WaitGroup, pendingBytesUpdated
 				maxBlockSize = l.Config.ThrottleBlockSize
 			}
 		}
-		var (
-			success bool
-			rpcErr  rpc.Error
-		)
-		err = cl.Client().CallContext(
-			ctx, &success, SetMaxDASizeMethod, hexutil.Uint64(maxTxSize), hexutil.Uint64(maxBlockSize),
-		)
-		if errors.Is(ctx.Err(), context.Canceled) {
-			// If the context was cancelled, our work is done and we expect an error here:
-			// So log it quietly and exit.
-			l.Log.Debug("DA throttling context cancelled")
-			return
-		}
-		if errors.As(err, &rpcErr) && eth.ErrorCode(rpcErr.ErrorCode()).IsGenericRPCError() {
-			l.Log.Error("SetMaxDASize rpc unavailable or broken, shutting down. Either enable it or disable throttling.", "err", err)
-			// We'd probably hit this error right after startup, so a short shutdown duration should suffice.
-			ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
-			defer cancel()
-			// Call StopBatchSubmitting in another goroutine to avoid deadlock.
-			go func() {
-				// Always returns nil. An error is only returned to expose this function as an RPC.
-				_ = l.StopBatchSubmitting(ctx)
-			}()
-			return
-		} else if err != nil {
-			l.Log.Error("SetMaxDASize rpc failed, retrying.", "err", err)
-			retryTimer.Reset(retryInterval)
-			return
-		}
-		if !success {
-			l.Log.Error("Result of SetMaxDASize was false, retrying.")
-			retryTimer.Reset(retryInterval)
+
+		// If we have configured DA update endpoints, distribute the configuration
+		if len(daClients) > 0 {
+			l.Log.Info("Distributing DA size configuration", "tx_size", maxTxSize, "block_size", maxBlockSize,
+				"endpoints", len(daClients))
+
+			var wg sync.WaitGroup
+			errors := make(chan error, len(daClients))
+			success := make(chan bool, len(daClients))
```

Comment:
you could work with a single `errors` channel and at the end just count the `nil`s.